### PR TITLE
Div 3660 time together

### PR DESCRIPTION
**Pull request by mounikahmcts**
Description: 

This page is to get confirmation on the time petitioner spent with his/her wife/husband since they separated once petitioner provides decision date and living apart date.
The page is content will vary based on below scenarios and dates given. It will follow 6 month separation rule.
1. Petitioner and respondent have not lived together since separating.
2. Petitioner and respondent have lived together since separating. 
3. Petitioner indicates the time lived together does not exceed the period of time presented to them 
4. Petitioner indicates the time lived together exceeds the period of time presented to them 

Fixes # https://tools.hmcts.net/jira/browse/DIV-3660

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/steps/grounds-for-divorce/lived-apart-since/index.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  getLiveTogetherPeriodRemainingDays(session) {
+    return this.getLivingTogetherDays(session) % constants.seven;
+  }
+
+  getSepStartDate(session) {
+    return moment().subtract(this.getSepYears(session), 'years')
+      .subtract(constants.six, 'months')
+      .format('DD MMMM YYYY');
+  }
+
```
Comment (gurpiarbassi) [inline app/steps/grounds-for-divorce/lived-apart-since/index.js]: extra blank line

Comment (borislavpetrovhmcts) [reply app/steps/grounds-for-divorce/lived-apart-since/index.js]: not a problem in javascript

Comment (mounikahmcts) [reply app/steps/grounds-for-divorce/lived-apart-since/index.js]: Yes lint was not giving any issues with extra lines. But removed extra lines now.

**Comment by mounikahmcts** [comment]
Closing this PR to rebase it against pr-362 changes and will reopen once it is ready

**Pull request closed without merging**